LDAR_Sim/testing/unit_testing/test_programs/test_method/test_survey_site.py

```diff
@@ -111,3 +121,140 @@ def test_000_simple_weather_finish_site(simple_method_values5):
     assert not last_survey
     assert travel_time == 1
     assert site_visited == 1
+
+
+@pytest.mark.parametrize(
```

Comment:
Nit: Could we move this out to a named fixture for each of these test cases?

Reply:
Fixtures don't work super well with parameterize but I will split this out another way